test/test_examples_2d_ape.jl

```diff
@@ -0,0 +1,31 @@
+module TestExamples2DAPE
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
+
+@testset "Acoustic Perturbation" begin
```

Comment:
Do the tests adher to the maximum time rules from https://trixi-framework.github.io/Trixi.jl/stable/testing/#Adding-new-tests?

Reply:
I get
```julia
julia> @time include("test/test_examples_2d_ape.jl")
...
  3.141501 seconds (4.93 M allocations: 264.763 MiB, 2.26% gc time)
```
for the second run on my machine, so I'd say yes.

Reply:
Sounds good!